src/layouts/docs/docs-layout.module.css

```diff
@@ -27,6 +27,10 @@
   overflow-y: auto;
   padding: 24px 24px 100px 24px;
   position: relative;
+
+  & :global(.__permalink-h) {
```

Comment:
Quick fix to hide permalinks as I wanted to take some screenshots for design review. I'll backlink to this comment in the `content styles` Asana task so this doesn't get buried 👍

Reply:
Maybe a `TODO` here would be helpful context too?